esrally/driver/runner.py

```diff
@@ -60,6 +60,7 @@ def register_default_runners():
     register_runner(track.OperationType.ClosePointInTime, ClosePointInTime(), async_runner=True)
     register_runner(track.OperationType.Sql, Sql(), async_runner=True)
     register_runner(track.OperationType.FieldCaps, FieldCaps(), async_runner=True)
+    register_runner(track.OperationType.Downsampling, Downsampling(), async_runner=True)
```

Comment:
Should the operation be considered administrative? Maybe I misunderstand but this is not something we would want multiple users doing, or performed multiple times, correct?

Reply:
This is something I am not sure. What is the criterion that we use to define an operation administrative? The expectation for this operation is that it is executed by means of an ILM action/policy (downsample the index when it reaches a certain size or after some amount of time). It might also be executed manually but I think those are more exceptional cases like for debugging purposes. So maybe these are good reasons to have the operation as an administrative operation?

Does it change something when the operation is executed?

Reply:
Note that this is an operation that we want to benchmark...so I guess if we make it administrative we don't want to retry.